tmo-monitor.py

```diff
@@ -31,9 +33,14 @@ def login_app(self):
     self.app_jar.set('lsid', login_request.cookies['lsid'], domain='192.168.12.1', path='/')
 
   def get_site_info(self):
-    if not self.app_jar:
-      self.login_app()
-    stat_request = requests.get('http://192.168.12.1/cell_status_app.cgi', cookies=self.app_jar)
+    try:
+      if not self.app_jar:
+        self.login_app()
+      stat_request = requests.get('http://192.168.12.1/cell_status_app.cgi', cookies=self.app_jar)
+    except:
+      print ("Could not query site info, exiting.")
+      sys.exit(2)
+
     stat_request.raise_for_status()
```

Comment:
I had wrapped all the gets as it seems the posts work without a connection to the modem. But the posts should fail without internet connection, so it makes sense to wrap all 7 requests. Done, thanks!

Reply:
Just for clarify, I meant that we want to `raise_for_status()` inside the try block so that when its exception is thrown that it is caught by the except block